pages/rating.html

```diff
@@ -0,0 +1,161 @@
+                <label for="rating2" title="2 stars">
+                  <i class="fa-solid fa-star"></i>
+                </label>
+
+                <input type="radio" id="rating1" name="rating" value="1" />
+                <label for="rating1" title="1 star">
+                  <i class="fa-solid fa-star"></i>
+                </label>
+              </section>
+            </div>
+
+            <div class="code-container">
+              <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Csection%2520class%253D%2522rate%2520size-4%2522%253E%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rating5%2522%2520name%253D%2522rating%2522%2520value%253D%25225%2522%2520%252F%253E%250A%2520%2520%253Clabel%2520for%253D%2522rating5%2522%2520title%253D%25225%2520stars%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fa-solid%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Flabel%253E%250A%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rating4%2522%2520name%253D%2522rating%2522%2520value%253D%25224%2522%2520%252F%253E%250A%2520%2520%253Clabel%2520for%253D%2522rating4%2522%2520title%253D%25224%2520stars%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fa-solid%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Flabel%253E%250A%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rating3%2522%2520name%253D%2522rating%2522%2520value%253D%25223%2522%2520%252F%253E%250A%2520%2520%253Clabel%2520for%253D%2522rating3%2522%2520title%253D%25223%2520stars%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fa-solid%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Flabel%253E%250A%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rating2%2522%2520name%253D%2522rating%2522%2520value%253D%25222%2522%2520%252F%253E%250A%2520%2520%253Clabel%2520for%253D%2522rating2%2522%2520title%253D%25222%2520stars%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fa-solid%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Flabel%253E%250A%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rating1%2522%2520name%253D%2522rating%2522%2520value%253D%25221%2522%2520%252F%253E%250A%2520%2520%253Clabel%2520for%253D%2522rating1%2522%2520title%253D%25221%2520star%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fa-solid%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Flabel%253E%250A%253C%252Fsection%253E"
+                style="
+                  height: 653px;
+                  border: 0;
+                  transform: scale(1);
+                  overflow: hidden;
+                "
+                sandbox="allow-scripts allow-same-origin"
```

Comment:
Minor suggestion: Try to avoid giving inline styles

Reply:
Thanks for the review. These styles are from **Carbon**, I can create a class containing other properties but not sure about height property as it is generated while creating iframe.